Documentation/operations/upgrade.rst

```diff
@@ -337,12 +337,6 @@ Annotations:
 
 * Cilium has bumped the minimal Kubernetes version supported to v1.13.0.
 
-Deprecated options
-~~~~~~~~~~~~~~~~~~
-
-* ``k8s-force-json-patch``: This option does not have any effect for
-  environments running Kubernetes >= 1.13. Marked for removal in Cilium v1.11.
-
```

Comment:
Why is this being removed? It looks like this is intended to be marked deprecated in 1.10 and removed in 1.11.

Reply:
I think this has been answered in the below comment.